src/playback/view/internal/abstractaudioresourceitem.h

```diff
@@ -66,11 +72,16 @@ class AbstractAudioResourceItem : public QObject
     void availableResourceListResolved(const QVariantList& resources);
 
 protected:
+    INJECT(muse::IGlobalConfiguration, globalConfiguration)
```

Comment:
so now can we move it to InputResourceItem?

Reply:
oh, I see that you use it in OutputResourceItem
I'll suggest duplicating it for them because if we decide to remove the action from one then this inject won't be useful for it